packages/pds-ember/app/styles/pds/theme/_light.scss

```diff
@@ -1,277 +1,79 @@
-$size--md:  scale.$base; // 16px (1rem)
-$size--lg:  scale.$lg-2; // 24px (1.5rem)
-$size--xl:  scale.$lg-5; // 36px (2.25rem)
-$size--2xl: scale.$lg-7; // 48px (3rem)
-
-
-// FIXME: "magic number" outside of defined scale
-//
-// DO NOT use this variable outside of this module!
-//
-// This private var is used to apply a 14px font size to theme variables
-// (i.e., variables defined in THIS file) that require it as part of design
-// specifications.  Because it is a value that doesn't exist in the modular
-// scale (see tokens/scale.scss), we need to keep it centralized to prevent
-// accidental use by consumers and to provide an upgrade path if designs are
-// modified in the future.
-$_size14: 0.875rem; // ~14px
-
-
-///----------------------------------------///
-/// CORE MEDIA/RESPONSIVE BREAKPOINTS
-///----------------------------------------///
-// Sass Only! (DO NOT allow client configuration)
-//
-// Even if we did allow client a11y configuration,
-// CSS custom properties can't be used in @media
-// queries.  Media Queries Level 5 spec will enable
-// this, but it's not yet available in any browser.
-
-// Tablets & Low-res PCs
-$media--sm: resolutions.$svga-width !default;
-// High-res PCs
-$media--md: resolutions.$hd-width !default;
-// Ultra High-res PCs
-$media--lg: resolutions.$fhd-width !default;
-
-
-///----------------------------------------///
-/// CORE TYPOGRAPHY
-///----------------------------------------///
-:root {
+$_black--a05: rgba(0, 0, 0, 0.05); //  5% black
+$_black--a06: rgba(0, 0, 0, 0.06); //  6% black
+$_black--a08: rgba(0, 0, 0, 0.08); //  8% black
+$_black--a10: rgba(0, 0, 0, 0.10); // 10% black
+$_black--a20: rgba(0, 0, 0, 0.20); // 20% black
+$_black--a25: rgba(0, 0, 0, 0.25); // 25% black
+
+// Set CSS Custom Properties
+@mixin styles {
+  /* [debug] #{$_module}@styles */
+  --pds-backgroundColor: #{color.$white};
+  --pds-color: #{color.$black};
+  --pds-focus-color: #{rgba(color.$blue-500, 0.5)};
+  // TYPOGRAPHY
+  // ----------
   // Per WCAG2.0 § 1.4.12 ("Text Spacing -- Level AA"),
   // these properties must be configurable in order
   // to meet requirements for Level AA compliance.
   //
   // For more info, see https://www.w3.org/WAI/WCAG21/quickref/#text-spacing
-  --pds-letterSpacing: 0.12em; // letter-spacing
-  --pds-lineHeight: 1.5;       // line-height (normal)
-  --pds-lineHeight--dense: 1.2; // line-height (dense)
-  --pds-paragraphSpacing: 2em; // margin-bottom (block-end margin)
-  --pds-wordSpacing: 0.16em;   // word-spacing
-}
-$fontFamily: font.$family-system-sans;
-$fontFamily--code: font.$family-system-mono;
-$fontSize: $size--md; // 16px (1rem)
-$fontSize--lg: scale.$lg--1; // 20px
-$fontSize--sm: scale.$sm--1; // 12px
-$fontWeight: font.weight(Regular);
-$letterSpacing: var(--pds-letterSpacing);
-$lineHeight: var(--pds-lineHeight);
-$lineHeight--dense: var(--pds-lineHeight--dense);
-$paragraphSpacing: var(--pds-paragraphSpacing);
-$wordSpacing: var(--pds-wordSpacing);
```

Comment:
This module has been updated so that PDS.Theme members are exported via `pds/theme/index.scss` (not the theme file).

As such, this theme file does not generate CSS on its own and its primary responsibility is to define the CSS custom properties via the `PDS.Theme.Light@styles` mixin.